### ISSUE TITLE
Add Support for Call Activity

### ISSUE THREAD

**Issue by philippfromme** [zeebe-io/zeebe-modeler#118]
__Is your feature request related to a problem? Please describe.__

Support for modeling `bpmn:CallActivities` elements is missing.

__Describe the solution you'd like__

- Add modeling support for call activities
- Make them available via replace menu, just like in the Camunda Modeler
- A CallActivity should be able to have a multi-instance marker

Zeebe Properties

* `callActivity` requires `extensionElements` with one `calledElement`
* `calledElement` has two optional attributes - _one of these should be set_
  * `processId` - the BPMN process Id of the called workflow
  * `processIdExpression` - the expression/variable name that resolves to the BPMN process Id
* optionally, a `callActivity` can have one or more input/output variable mappings (`extensionElements` with `ioMapping`)

```xml
<bpmn:callActivity id="Task_09pdy54">
    <bpmn:extensionElements>
        <zeebe:calledElement 
            processId="myProcess" 
            processIdExpression="myProcessVariable" />
     </bpmn:extensionElements>
</bpmn:callActivity>
```

__Describe alternatives you've considered__

/

__Additional context__

![image](https://user-images.githubusercontent.com/9433996/65575844-07670000-df71-11e9-95da-ea4a759687b1.png)

**Comment by pinussilvestrus** [comment]
We will have to discuss, which zeebe related properties have to be added here. @menski can give further information on this.

**Comment by oguzeroglu** [comment]
Based on this: https://docs.zeebe.io/bpmn-workflows/bpmn-coverage.html#bpmn-coverage

The call activities are not implemented by Zeebe, not exactly sure how fresh is that document though.

**Comment by pinussilvestrus** [comment]
Will be supported via the next Zeebe v0.22.0 release, right @menski?

**Comment by saig0** [comment]
Correct :rocket:

**Comment by pinussilvestrus** [comment]
Support will be added by #115, waiting for zeebe related properties information to work on this issue.

**Comment by saig0** [comment]
* `callActivity` requires `extensionElements` with one `calledElement`
* `calledElement` has two optional attributes - one of these should be set
  * `processId` - the BPMN process Id of the called workflow
  * `processIdExpression` - the expression/variable name that resolves to the BPMN process Id
* optionally, a `callActivity` can have one or more input/output variable mappings (`extensionElements` with `ioMapping`)

```xml
<bpmn:callActivity id="Task_09pdy54">
    <bpmn:extensionElements>
        <zeebe:calledElement 
            processId="myProcess" 
            processIdExpression="myProcessVariable" />
     </bpmn:extensionElements>
</bpmn:callActivity>
```

**Comment by pinussilvestrus** [comment]
Thanks for clarifying 👍

**Comment by pinussilvestrus** [comment]
Closed via #131
